pkg/policy/rule_test.go

```diff
@@ -638,6 +638,82 @@ func (ds *PolicyTestSuite) TestL3Policy(c *C) {
 	c.Assert(err, Not(IsNil))
 }
 
+// Tests the restrictions of combining certain label-based L3 and L4 policies.
+// This ensures that the user is informed of policy combinations that are not
+// implemented in the datapath.
+func (ds *PolicyTestSuite) TestEgressRuleRestrictions(c *C) {
+
+	fooSelector := []api.EndpointSelector{
+		api.NewESFromLabels(labels.ParseSelectLabel("foo")),
+	}
+
+	// Cannot combine ToEndpoints and ToCIDR
+	apiRule1 := api.Rule{
+		EndpointSelector: api.NewESFromLabels(labels.ParseSelectLabel("bar")),
+		Egress: []api.EgressRule{
+			{
+				ToCIDR: []api.CIDR{
+					"10.1.0.0/16",
+					"2001:dbf::/64",
+				},
+				ToEndpoints: fooSelector,
+			},
+		},
+	}
+
+	err := apiRule1.Sanitize()
+	c.Assert(err, Not(IsNil))
+
+	// Cannot combine ToEndpoints and ToPorts
```

Comment:
Why not?

Reply:
Because I haven't done the work for egress label-dependent L4. I wanted to follow that up in another PR.

Reply:
Create GH issue pleasae and add to the comments

Reply:
Cool, the datapath is 🙆

Reply:
FYI - I have a follow-up branch for this coming very soon.